extensions/spark/kyuubi-spark-authz/pom.xml

```diff
@@ -336,6 +336,18 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>hudi</id>
+            <dependencies>
+                <dependency>
+                    <groupId>org.apache.hudi</groupId>
+                    <artifactId>hudi-spark${hudi.spark.binary.version}-bundle_2.12</artifactId>
+                    <version>${hudi.version}</version>
+                    <scope>test</scope>
+                </dependency>
+            </dependencies>
+        </profile>
```

Comment:
Avoid import hudi jar when enable scala 2.13 so add this profile cc @pan3793 @yaooqinn 
We can remove this after hudi support scala 2.13

Reply:
does Hudi community have a JIRA ticket to track the progress of support for Scala 2.13? let's leave some comments here with a brief explanation of the background

Reply:
Done